packages/cf-component-wizard/README.md

```diff
@@ -0,0 +1,99 @@
+# cf-component-wizard
+
+> CloudFlare Wizard Component
+
+## Installation
+
+```sh
+$ npm install cf-component-wizard
+```
+
+## Usage
+
+```js
+const React = require('react');
+const {render} = require('react-dom');
+const Wizard = require('../../src/index');
+const {Button} = require('cf-component-button');
+
+const {
+  GatewayDest,
+  GatewayProvider
+} = require('react-gateway');
+
+class Component extends React.Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      activeStep: 'step1',
+      isOpen: false
+    };
+
+    this.handleRequestOpen = this.handleRequestOpen.bind(this);
+    this.handleRequestClose = this.handleRequestClose.bind(this);
```

Comment:
Should we just get rid of this and do the binds where the function is used? Either that or change line 58 to do the same as this.

Only reason why I'm suggesting it is because I'm more use to the pattern of binding before passing down.